### PR TITLE
Add cli flag to enable root volume encryption

### PR DESCRIPTION
**Pull request by muraee**
**What this PR does / why we need it**:
Enables setting KMS encryption key for nodepool root volume encryption.

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

**Comment by enxebre** [comment]
thanks!
/approve
PTAL @bryan-cox

File: cmd/cluster/core/create.go [hunk 1]
```diff
@@ -120,21 +120,23 @@ type KubevirtPlatformCreateOptions struct {
...
+	PublicZoneID            string
+	Region                  string
+	RootVolumeIOPS          int64
+	RootVolumeSize          int64
+	RootVolumeType          string
+	RootVolumeEncryptionKey string
+	EndpointAccess          string
+	Zones                   []string
+	EtcdKMSKeyARN           string
+
```
Comment (bryan-cox) [inline cmd/cluster/core/create.go:138]: nit: remove the new line so all the struct variables line up.

**Comment by bryan-cox** [comment]
Looks good to me. Just had one really minor nit. 

Does this have a corresponding Jira ticket? If so, we should probably reference that in the PR description.

**Pull request merged**